api/v1/openapi.yaml

```diff
@@ -1429,6 +1432,86 @@ definitions:
           direct-server-return:
             description: Perform direct server return
             type: boolean
+  ProxyStatus:
```

Comment:
Should `ProxyStatus` have a `Status` type subfield? This would make its status more consistent with the other fields in this API.

Reply:
We don't really have anything to report right now. I agree that it makes sense when we have that.